docs/root/intro/arch_overview/outlier.rst

```diff
@@ -10,6 +10,17 @@ such as consecutive failures, temporal success rate, temporal latency, etc. Outl
 form of *passive* health checking. Envoy also supports :ref:`active health checking
 <arch_overview_health_checking>`. *Passive* and *active* health checking can be enabled together or
 independently, and form the basis for an overall upstream health checking solution.
+Outlier detection is part of :ref:`cluster configuration <envoy_api_msg_cluster.OutlierDetection>`, 
+but it needs network filters to report errors, timeouts, resets. Currently the following filters support
```

Comment:
"network or HTTP filters" ?

Reply:
AFAIK only HTTP router, tcp_proxy and redis_proxy provide feedback to outlier detector. The latter two do not really talk http, so I called them "network".